modules/mesh-task/outputs.tf

```diff
@@ -14,3 +14,6 @@ output "task_tags" {
   value = aws_ecs_task_definition.this.tags_all
 }
 
+output "service_token_secret_arn" {
+  value = var.acls ? aws_secretsmanager_secret.service_token[0].arn : ""
+}
```

Comment:
I don't know about documenting it. We don't have versioned ECS docs, but we could backport a doc change to the release/1.11.x branch, and then it would show there?

Reply:
I also updated the description of the `execution_role` input variable to explain this.